taskcluster/tc-tests-utils.sh

```diff
@@ -1443,7 +1440,7 @@ do_deepspeech_nodejs_build()
       RASPBIAN=${SYSTEM_RASPBIAN} \
       TFDIR=${DS_TFDIR} \
       NODE_ABI_TARGET=--target=$electron \
-      NODE_DIST_URL=--disturl=https://electronjs.org/headers \
+      NODE_DIST_URL=--disturl=https://reuben.github.io/electronjs_headers \
```

Comment:
I'll remove that in #2737

Reply:
Oh, I should have looked there first. I guess if that's ready to land I should just remove this change from here.